.github/workflows/build.yml

```diff
@@ -0,0 +1,15 @@
+name: build
+
+on:
+  release:
+    types: [ released ]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@v1
+    with:
+      upload_to_pypi: ${{ github.event_name == 'released'}}
```

Comment:
Thanks for updating this workflow. Should this be `release` instead of `released`? I attempted to look through the github docs but failed to find anything definitive.